packages/server/src/modules/zoom/service.ts

```
@@ -0,0 +1,58 @@
+import axios from 'axios';
+
+import { service } from '@application';
+
+const Authorization = Buffer.from(
+  `${process.env.ZOOM_MEETING_SDK_KEY_OR_CLIENT_ID}:${process.env.ZOOM_MEETING_SDK_SECRET_OR_CLIENT_SECRET}`,
+).toString('base64');
+
+const authorizeUserByCode = async (code: string, redirectUri: string) => {
+  const accessToken = await axios
+    .postForm(
+      'https://zoom.us/oauth/token',
+      {
+        code,
+        grant_type: 'authorization_code',
+        redirect_uri: redirectUri,
+      },
+      {
+        headers: {
+          Authorization: `Basic ${Authorization}`,
+          'Content-Type': 'application/x-www-form-urlencoded',
+        },
+        method: 'POST',
+      },
+    )
+    .then(data => data.data.access_token)
+    .catch(console.log);
```

Don't catch error here in such way. Let it propagate to a caller.